pkg/movingaverage/moving_average_test.go

```diff
@@ -90,19 +90,19 @@ type testCase struct {
 
 func (t *testMovingAvg) TestMovingAvg(c *C) {
 	var empty float64 = 0
-	data := []float64{120, 130, 140, 150, 145, 136, 121, 132, 145, 156, 148, 157, 175}
```

Comment:
Can the old test case be kept?

Reply:
it work well,but i think it should be simple for verify by manual

Reply:
add some test with jitter? such as [1,1,1,1,5,1,1,1]?